pkg/client/client.go

```diff
@@ -86,6 +89,56 @@ func (c *ImmuClient) Get(keyReader io.Reader) (*schema.Item, error) {
 	return result, err
 }
 
+// SafeGet ...
+func (c *ImmuClient) SafeGet(keyReader io.Reader) (*schema.Item, bool, error) {
```

Comment:
@mmeloni Do you think it is ok to return an `Item` or should we create a dedicated struct (for simple Get an `Item` is returned, so probably if we do not want to expose it we need to change everywhere in this file).